src/components/StoryPageTimeline.js

```diff
@@ -104,6 +110,30 @@ const StoryPageTimeline = (props) => {
         >
           {votes} Votes
         </div>
+        {showLog && (
+          <>
+            <Modal
+              content={
+                <>
+                  <div>
```

Comment:
try not to use too many empty divs that don't even have classes. Titles are display block elements, they don't need to be encapsulated in divs